src/search/pdbs/pattern_database.cc

```diff
@@ -70,7 +73,10 @@ PatternDatabase::PatternDatabase(
     const TaskProxy &task_proxy,
     const Pattern &pattern,
     bool dump,
-    const vector<int> &operator_costs)
+    const vector<int> &operator_costs,
+    bool compute_plan,
+    const shared_ptr<utils::RandomNumberGenerator> &rng,
```

Comment:
Is the rng still needed?

Reply:
(Applies to many places.)

Reply:
Yes, when computing a regular plan, we randomly choose one operator from the set of equivalent ones. This could be done in a post processing step, though, but I thought it would be nicer to have the plan computation "done" when it's done.